en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/C/Contagion.md

```diff
@@ -33,8 +33,8 @@ After three failed saves, the disease lasts for the duration and the creature st
 A [Greater Restoration](#Greater_Restoration_greater_restoration) spell or similar effect also ends the disease.
 
 <!-- spell-checker:words Mindfire -->
-- **Blinding Sickness.**
-  The target is [blinded](#Conditions_blinded) and has disadvantage on Wisdom checks and Wisdom saving throws.
```

Comment:
This reduces the usefulness of this specific option considerably. Maybe we can consider something similar to blinding the target? Could be the same (_nullfiied: sight_) or could be something else.

Also, this changes the metadata entry `compared_to_wotc_srd_5.1` to
```md
[_metadata_:compared_to_wotc_srd_5.1]:- "mechanics_different_wording_different"
```